### DISCUSSION TITLE
July 18 Friday HackHour: Machine-learning (pytorch) cloud-classification using hyperspectral spectral data

### DISCUSSION THREAD

**Discussion by eeholmes** [nmfs-openscapes/nmfs-openscapes#28] [HackHours]
@nmfs-openscapes/2024-science-teams 

If you have not registered for the Zoom link, do so here (you only need to do once) [Register here](https://zoom.us/meeting/register/Om99ecCMSrqepyDVDJ7bBA) for video links. Open to non-NOAA folks too.

Guest speaker Sean Foley (NASA) will give an example of using neural networks for a classification task with pytorch.

Background: Cloud masking refers to the task of assigning a binary value to each pixel in a satellite image, according to whether or not that location is covered by a cloud. It is essential to many other data products. For non-atmospheric scientists, this is usually useful to know what pixels to ignore, or ‘mask out’, in processing. For atmospheric scientists, it is often the opposite: we may exclusively wish to study the clouds. In this session, we will see how to use a Simple Multi-Layer Perceptron (MLP) to perform this task.

We'll be using pytorch. This is the most popular machine learning library among researchers as it represents a good balance between flexibility and ease-of-use.
